cli/node_account/cli.py

```diff
@@ -58,3 +61,29 @@ def get(address, node_url):
         sys.exit(FAILED_EXIT_FROM_COMMAND_CODE)
 
     print_result(result=result)
+
+
+@node_account_commands.command('transfer-tokens-from-unfrozen-to-frozen')
```

Comment:
`from frozen to unfrozen `, change it's everywhere, please.

Reply:
@Alladin9393, done.